InsiteTeamTask.Data/Repositories/Implementations/GamesRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using InsiteTeamTask.Data.DTOs;
+using InsiteTeamTask.Data.MockData;
+using InsiteTeamTask.Data.Repositories.Interfaces;
+
+namespace InsiteTeamTask.Data.Repositories.Implementations
+{
+    internal class GamesRepository :Repository, IGamesRepository
+    {
+        public GamesRepository(IMapper mapper, IDataService dataService) : base(mapper, dataService)
+        {
+        }
+
+        public IEnumerable<GameDto> GetAllGames()
+        {
+            return Mapper.Map<IEnumerable<GameDto>>(DataService.Games());
```

Comment:
I like the splitting of data into separate repositories, but when their only use is to funnel Games, Members and Products back into a single `ListService` it seems kind of pointless. This pattern is actually part of what we were looking for, but you haven't followed through with individual services and controllers.